code/modules/research/designs/mechfabricator_designs.dm

```diff
@@ -672,12 +672,12 @@
 	construction_time = 100
 	category = list("Exosuit Equipment")
 
-/datum/design/mech_taser
-	name = "Exosuit Weapon (PBT \"Pacifier\" Mounted Taser)"
-	id = "mech_taser"
+/datum/design/mech_disabler
+	name = "Exosuit Weapon (CH-DS \"Peacemaker\" Disabler)"
+	id = "mech_disabler"
```

Comment:
The old mech disabler still exists in the code, and now there is 2 disablers that robotics can print that do the same thing? 
(Line 906)
Could probably just change the old disabler build path instead of changing the taser construction, and just remove the taser construction instead.